README.md

```diff
@@ -101,46 +113,77 @@ Note: Omit `poetry run` in all example commands going forward if you've installe
-These can be viewed by opening `web/index.html` in a web browser. E.g., `firefox web/index.html`. 
+After a successful benchmark run, static HTML pages are generated that
+display scores on benchmarks and tests. These can be viewed by opening
+`web/index.html` in a web browser. E.g., `firefox web/index.html`.
+
+Note that the HTML that ModelBench produces is an older version than is available
+on [the website](https://ailuminate.mlcommons.org/). Over time we'll simplify the
+direct ModelBench output to be more straightforward and more direclty useful to
+people independently running ModelBench.
+
+### Using the journal
+
+As modelbench runs, it logs each important event to the journal. That includes
+every step of prompt processing. You can use that to extract most information
+that you might want about the run. The journal is a zstandard-compressed JSONL
+file, meaning that each line is a valid JSON object.
 
-If you would like to dump the raw scores, you can do:
+There are many tools that can work with those files. In the example below, we
+use [jq](https://jqlang.github.io/jq/, a JSON swiss army knife. For more
+information on the journal, see [the documentation](docs/run-journal.md).
+
+To dump the raw scores, you could do something like this
 
 ```shell
-poetry run modelbench grid -m 10 > scoring-grid.csv
+zstd -d -c $(ls run/journals/* | tail -1)  | jq -rn ' ["sut", "hazard", "score", "reference score"], (inputs | select(.message=="hazard scored") | [.sut, .hazard, .score, .reference]) | @csv'
 ```
 
-To see all raw requests, responses, and annotations, do:
+That will produce CSV for each hazard scored, as well as showing the reference
+score for that hazard.
+
+Or if you'd like to see the processing chain for a specific prompt, you could do:
 
 ```shell
-poetry run modelbench responses -m 10 response-output-dir
+zstd -d -c $(ls run/journals/* | tail -1)  | jq -r 'select(.prompt_id=="airr_practice_1_0_41321")'
 ```
-That will produce a series of CSV files, one per Harm, in the given output directory. Please note that many of the
-prompts may be uncomfortable or harmful to view, especially to people with a history of trauma related to one of the
-Harms that we test for. Consider carefully whether you need to view the prompts and responses, limit exposure to
-what's necessary, take regular breaks, and stop if you feel uncomfortable. For more information on the risks, see
-[this literature review on vicarious trauma](https://www.zevohealth.com/wp-content/uploads/2021/08/Literature-Review_Content-Moderators37779.pdf).
+
+That should output a series of JSON objects showing the flow from `queuing item`
+to `item finished`.
+
+**CAUTION**: Please note that many of the prompts may be uncomfortable or
+harmful to view, especially to people with a history of trauma related to
+one of the Harms that we test for. Consider carefully whether you need
```

Comment:
I think harm doesn't need to be capitalized since it's not a canonical concept within our system.